### PR TITLE
Atlas fixes

### PR DESCRIPTION
**Pull request by ColdEvul**
**When merged this pull request will:**
- FIXED References being wrongly implemented
- CHANGED Loadouts to fit all objects

### CODE REVIEW

File: cScripts/Loadouts/CfgLoadouts_Bravo_Atlas.hpp [hunk 1]
```diff
@@ -153,10 +149,38 @@ class Cav_B_B_Atlas_Medic_CombatMedic_F : Cav_B_Bravo_Atlas_base_F {
...
+
+        "SmokeShell",2,
+        "SmokeShellGreen",2,
+        "ACE_Chemlight_UltraHiOrange",6,
+        "B_IR_Grenade",6
+    };
     items[] += {
+
+        // Personal Medical Equipment
+        "ACE_elasticBandage",10,
```
Comment (longlostblake) [inline cScripts/Loadouts/CfgLoadouts_Bravo_Atlas.hpp:172]: Not sure why we have elastics here?

Comment (ColdEvul) [reply cScripts/Loadouts/CfgLoadouts_Bravo_Atlas.hpp:172]: Else they had none

Comment (longlostblake) [reply cScripts/Loadouts/CfgLoadouts_Bravo_Atlas.hpp:172]: Shouldn't they be ACE_QuikClot ?

Comment (ColdEvul) [reply cScripts/Loadouts/CfgLoadouts_Bravo_Atlas.hpp:172]: depr yes it should...

Comment (ColdEvul) [reply cScripts/Loadouts/CfgLoadouts_Bravo_Atlas.hpp]: Fixed

**Comment by longlostblake** [comment]
Checked.

**Pull request merged**